config/config.go

```diff
@@ -11,7 +11,8 @@ type Configuration struct {
 	Port            int                `mapstructure:"port"`
 	AdminPort       int                `mapstructure:"admin_port"`
 	DefaultTimeout  uint64             `mapstructure:"default_timeout_ms"`
-	CacheURL        string             `mapstructure:"prebid_cache_url"`
+	CacheUrl        string             `mapstructure:"prebid_cache_url"`
+	Macros          string             `mapstructure:"macros"`
```

Comment:
I think we should consolidate the "cache configs" in a struct. For example, something like:

```
{
  "cache": {
    "host": "prebidcache.net",
    "query": "uuid=%PBS_CACHE_UUID%"
  }
}
```

It'll help keep the config organized, and we'll have a scalable structure if people want to add config options for the scheme, port, etc, later.